.eslintrc.js

```diff
@@ -128,14 +121,12 @@ module.exports = {
     "implicit-arrow-linebreak": 0,
     // XXX Switch back to indent once mozilla-central has decided what it is using.
     "indent": 0,
-    "indent-legacy": ["error", 2, {"SwitchCase": 1}],
     "init-declarations": 0,
-    "jsx-quotes": [2, "prefer-double"],
     "line-comment-position": 0,
     "lines-around-comment": ["error", {
       "allowClassStart": true,
-      "allowObjectStart": true,
-      "beforeBlockComment": true
+      "beforeBlockComment": false,
+      "allowBlockStart": true,
```

Comment:
I still don't understand why you can't just add the one `"allowBlockStart": true` as suggested in https://github.com/mozilla/activity-stream/pull/5084#discussion_r289472340 instead of deleting `"allowObjectStart": true,` and switching `beforeBlockComment` to `false`.

If I checkout "591bbf5f - Install eslint/prettier package versions matching mozilla-central", add `"allowBlockStart": true,`, run `--fix` and commit, apply "e2e22a0d - Rejigger various eslint-disable-* to play nice with prettier"…

`npm run lint` is happy and there's no differences (other than the `.eslintrc.js` with the one added line).

To be clear, `npm run eslint-check` exits failure 2 listing `- lines-around-comment` but that happens on the latest version of the PR as well.

Reply:
Oh oops. I see where I did differently. I see one difference:
```diff
diff --git a/test/unit/lib/Tokenize.test.js b/test/unit/lib/Tokenize.test.js
--- a/test/unit/lib/Tokenize.test.js
+++ b/test/unit/lib/Tokenize.test.js
@@ -76,16 +76,17 @@ describe("TF-IDF Term Vectorizer", () => {
         expected: {
           easy: [269, 0.6786999710383944],
           care: [153, 0.7344156515982504],
         },
       },
       {
         input: "这个空间故意留空。",
         expected: {
+
           /* This space is left intentionally blank. */
         },
       },
     ];
     let checkTokenGeneration = tc => {
       describe(`${tc.input} should have only vocabulary tokens`, () => {
         let actual = toksToTfIdfVector(tokenize(tc.input), vocab_idfs);
 
```

Reply:
Oh no nevermind again. This is with your deleting `"allowObjectStart": true`

Reply:
So I just talked to Victor, and it sounds like he's going to be disabling all rules that eslint-config-check flags as "possibly conflicting" by eslint-config-prettier at the top-level, which sounds reasonable to me.  So I'm inclined to just get rid of `lines-around-comment` entirely to keep us in line with that.  Any thoughts?

Reply:
To be clear, you've already effectively turned off the rule by by setting `"beforeBlockComment": false,`

If we don't want any conflicts showing up, then removing the rule is the only thing we can do then.

Reply:
Right; I guess what I was really trying to ask was "is there any compelling reason I should push back on this proposed policy"?

Reply:
Looking at when we enabled lines-around-comment https://github.com/mozilla/activity-stream/commit/b330de028ff62d1a4e6794f9e09a6c5cacfa51fd The main change was making our comment types more consistent -- basically block comments above blocks/functions and inline comments inline.

We can always just `nit` if someone doesn't add a line before block comments or uses the different type of comment.

Reply:
The irony being that prettier is all about getting rid of nits.  Another alternative would be just not to worry about comment-type consistency.

Anyway, I'll go ahead and get rid of lines-around-comment.